{"uuid":"c3a31bab-1a84-4519-a41f-4ef1e6cae36b","name":"TestCRM","children":["ab1694ec-0a07-46ec-9c3b-7eb964b896d0"],"befores":[{"name":"setExtent","status":"passed","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1612440064571,"stop":1612440064646}],"afters":[{"name":"endReport","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Sending the email to the following server failed : smtp.gmail.com:465","trace":"org.apache.commons.mail.EmailException: Sending the email to the following server failed : smtp.gmail.com:465\r\n\tat org.apache.commons.mail.Email.sendMimeMessage(Email.java:1469)\r\n\tat org.apache.commons.mail.Email.send(Email.java:1496)\r\n\tat BasePage.BasePage.SendEmail(BasePage.java:813)\r\n\tat BasePage.BasePage.endReport(BasePage.java:768)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.lang.reflect.Method.invoke(Unknown Source)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)\r\n\tat org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)\r\n\tat org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)\r\n\tat org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:617)\r\n\tat org.testng.TestRunner.afterRun(TestRunner.java:847)\r\n\tat org.testng.TestRunner.run(TestRunner.java:588)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:384)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:286)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1140)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1069)\r\n\tat org.testng.TestNG.run(TestNG.java:1037)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\nCaused by: javax.mail.AuthenticationFailedException: 535-5.7.8 Username and Password not accepted. Learn more at\n535 5.7.8  https://support.google.com/mail/?p=BadCredentials b16sm4838280qkh.21 - gsmtp\n\r\n\tat com.sun.mail.smtp.SMTPTransport$Authenticator.authenticate(SMTPTransport.java:932)\r\n\tat com.sun.mail.smtp.SMTPTransport.authenticate(SMTPTransport.java:843)\r\n\tat com.sun.mail.smtp.SMTPTransport.protocolConnect(SMTPTransport.java:748)\r\n\tat javax.mail.Service.connect(Service.java:388)\r\n\tat javax.mail.Service.connect(Service.java:246)\r\n\tat javax.mail.Service.connect(Service.java:195)\r\n\tat javax.mail.Transport.send0(Transport.java:254)\r\n\tat javax.mail.Transport.send(Transport.java:124)\r\n\tat org.apache.commons.mail.Email.sendMimeMessage(Email.java:1459)\r\n\t... 27 more\r\n"},"stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1612440144122,"stop":1612440148200}],"links":[],"start":1612440064441,"stop":1612440148406}